src/components/UserList/UserList.tsx

```diff
@@ -0,0 +1,205 @@
+import { Box, Table, TableBody, TableCell, TableContainer, TablePagination, TableRow } from '@mui/material';
+import { ChangeEvent, MouseEvent, useEffect, useState } from 'react';
+import { Data, Order } from './UserList.types';
+import Checkbox from '@mui/material/Checkbox';
+import { UserListHead } from './UserListHead';
+import { UserListToolbar } from './UserListToolbar';
+
+function descendingComparator<T>(firstValue: T, secondValue: T, orderBy: keyof T) {
+  if (secondValue[orderBy] < firstValue[orderBy]) {
+    return -1;
+  }
+  if (secondValue[orderBy] > firstValue[orderBy]) {
+    return 1;
+  }
+  return 0;
+}
+function getComparator<Key extends keyof Data>(
+  order: Order,
+  orderBy: Key
+): (firstValue: { [key in Key]: number | string }, secondValue: { [key in Key]: number | string }) => number {
+  return order === Order.DESC
+    ? (firstValue, secondValue) => {
+        return descendingComparator(firstValue, secondValue, orderBy);
+      }
+    : (firstValue, secondValue) => {
+        return -descendingComparator(firstValue, secondValue, orderBy);
+      };
+}
+
+export function UserList() {
+  const [order, setOrder] = useState<Order>(Order.ASC);
+  const [orderBy, setOrderBy] = useState<keyof Data>('id');
+  const [selected, setSelected] = useState<readonly number[]>([]);
+  const [page, setPage] = useState(0);
+  const [rows, setRows] = useState<Data[]>([]);
+  const defaultRowsPerPage = 10;
+  const [rowsPerPage, setRowsPerPage] = useState(defaultRowsPerPage);
+  const pixelHeightPerRow = 53;
```

Comment:
does it trigger eslint magic number rule?

Reply:
yes

Reply:
We will get rid of this rule as it triggers too many magic numbers in this project. Please add magic number to disabled rules